test/integration/rules/aria-allowed-role/aria-allowed-role.html

```diff
@@ -7,7 +7,6 @@
 <section id="pass-section-role-doc-bib" role="doc-bibliography"></section>
 <ul><li id='pass-li-role-doc-biblioentry' role="doc-biblioentry"></li></ul>
 <aside id='pass-aside-doc-example' role='doc-example'></aside>
-<div id='pass-div-has-any-role' role='divAnyRoleEvenInvalid'></div>
```

Comment:
Instead of removing, I suggest you put in a valid role.

Reply:
Done. Added it back again. This diff only watches this line, so comment is not going stale.